web/config/navItems.js

```diff
@@ -1,3 +1,4 @@
 export const MAIN_NAV_ITEMS = [
   { path: '/products', display: 'Products' }
+  , { path: '/tables', display: 'Tables' }
```

Comment:
remove

Reply:
also re-remove all the static/ css and js and .map files from git. they should be gitignored

Reply:
and removed the entire server/public folder